staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/inner/metadataname/doc.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:validation-gen=*
+// +k8s:validation-gen-scheme-registry=k8s.io/code-generator/cmd/validation-gen/testscheme.Scheme
+
+// Package inner contains test types for testing inner field validation tags.
+package inner
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/code-generator/cmd/validation-gen/testscheme"
+)
+
+var localSchemeBuilder = testscheme.New()
+
+type T1 struct {
+	// +k8s:inner(Name)=+k8s:validateFalse="inner T1.ObjectMeta.Name"
```

Comment:
Oh, that's so nice.

Reply:
Also - last chance to find a word better than "inner" :)

Reply:
I've change this to now be s/inner/subField

eg:
// +k8s:subField(Name)=+k8s:validateFalse="inner T1.ObjectMeta.Name"

Reply:
changed this to subfield now (from subField)